### PR TITLE
THIRD_PARTY_LICENSES.txt

### PR DESCRIPTION
**Pull request by shihchang**
update THIRD_PARTY_LICENSES.txt

### CODE REVIEW

File: THIRD_PARTY_LICENSES.txt [hunk 1]
```diff
@@ -0,0 +1,11239 @@
...
+means all the source code for all modules it contains, plus any
+associated interface definition files, plus the scripts used to control
+compilation and installation of the executable. However, as a special
+exception, the source code distributed need not include anything that is
+normally distributed (in either source or binary form) with the major
+components (compiler, kernel, and so on) of the operating system on
+which the executable runs, unless that component itself accompanies the
+executable.
+
+If distribution of executable or object code is made
```
Comment (markxnelson) [inline THIRD_PARTY_LICENSES.txt:11199]: this should not be here

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: THIRD_PARTY_LICENSES.txt (hunk 1 above)
Comment (shihchang) [reply THIRD_PARTY_LICENSES.txt]: removed the chunk of code at the bottom. replaced it to the content of that licenseText.

File: known_licenses.txt [hunk 2]
```diff
@@ -0,0 +1,72 @@
+babel-plugin-syntax-class-properties@6.13.0 MIT
```
Comment (markxnelson) [inline known_licenses.txt:1]: this file cannot be checked int

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: known_licenses.txt (hunk 2 above)
Comment (shihchang) [reply known_licenses.txt]: removed

**Review by markxnelson — APPROVED** [review]

**Pull request merged**